src/xdocs/config_annotation.xml

```diff
@@ -141,59 +141,61 @@ public String getNameIfPresent() { ... }
-    value=&quot;false&quot;/&gt;
-  &lt;property name=&quot;allowSamelineParameterizedAnnotation&quot; value=&quot;false&quot;/&gt;
+&lt;property name=&quot;allowSamelineMultipleAnnotations&quot; value=&quot;true&quot;/&gt;
+&lt;property name=&quot;allowSamelineSingleParameterlessAnnotation&quot;
+value=&quot;false&quot;/&gt;
+&lt;property name=&quot;allowSamelineParameterizedAnnotation&quot; value=&quot;false&quot;/&gt;
 &lt;/module&gt;
-        </source>
-        <p>
-          Example to allow one single parameterless annotation on the same line
-        </p>
-        <source>
-@Override public int hashCode() { ... } // no violations
-@SuppressWarnings("deprecation") public int foo() { ... } // violation
-        </source>
-        <p>
-          Use the following configuration:
-        </p>
-        <source>
+          </source>
+          <p>
+              Example to allow multiple annotations on the same line
+          </p>
+          <source>
+              @SuppressWarnings("deprecation") @Mock DataLoader loader; // no violations
+          </source>
+          <p>
+              Use the following configuration to allow one single parameterless
+              annotation on the same line:
+          </p>
+          <source>
 &lt;module name=&quot;AnnotationLocation&quot;&gt;
-  &lt;property name=&quot;allowSamelineMultipleAnnotations&quot; value=&quot;false&quot;/&gt;
-  &lt;property name=&quot;allowSamelineSingleParameterlessAnnotation&quot;
-    value=&quot;true&quot;/&gt;
-  &lt;property name=&quot;allowSamelineParameterizedAnnotation&quot; value=&quot;false&quot;/&gt;
+&lt;property name=&quot;allowSamelineMultipleAnnotations&quot; value=&quot;false&quot;/&gt;
+&lt;property name=&quot;allowSamelineSingleParameterlessAnnotation&quot;
+value=&quot;true&quot;/&gt;
+&lt;property name=&quot;allowSamelineParameterizedAnnotation&quot; value=&quot;false&quot;/&gt;
 &lt;/module&gt;
-        </source>
-        <p>
-          Example to allow only one and only parameterized annotation on the same line
-        </p>
-        <source>
-@SuppressWarnings("deprecation") DataLoader loader; // no violations
-@Mock DataLoader loader; // violation
-        </source>
-        <p>
-          Use the following configuration:
-        </p>
-        <source>
+          </source>
+          <p>
+              Example to allow one single parameterless annotation on the same line
+          </p>
+          <source>
+              @Override public int hashCode() { ... } // no violations
+              @SuppressWarnings("deprecation") public int foo() { ... } // violation
+          </source>
+          <p>
+              Use the following configuration to allow only one and
+              only parameterized annotation on the same line:
+          </p>
+          <source>
 &lt;module name=&quot;AnnotationLocation&quot;&gt;
-  &lt;property name=&quot;allowSamelineMultipleAnnotations&quot; value=&quot;false&quot;/&gt;
```

Comment:
Done

Reply:
I didn't add a separate section for default configuration because it will duplicate the same. So I mentioned about the default in the relevant existing config.

Reply:
@PrabodDunuwila We should have default config. It shows users default behavior versus customized behavior.